package.json

```diff
@@ -76,6 +77,7 @@
     "typescript": "^4.8.2"
   },
   "dependencies": {
+    "blakejs": "^1.2.1",
```

Comment:
Should we be fixing this version for a crypto dependency like this? Curious to your thoughts

Reply:
Good question! To me it makes sense to do this.